data/templated/example.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<query_templates>
+   <query_template>
+      <name>GetOrder</name>
+      <query><![CDATA[SELECT NO_O_ID FROM new_order WHERE NO_D_ID = ? OR NO_D_ID = ? ORDER BY NO_O_ID ASC]]></query>
+      <parameter_types>4,4</parameter_types>
```

Comment:
Are the string names for these unambiguous enough that we could use them instead of integers?
I find that to be somewhat harder to read.

Reply:
Done! It now parses the String from the sql.Types into an integer internally.